test/sqlite3_monkey_patch.rb

```diff
@@ -0,0 +1,23 @@
+require 'active_record/connection_adapters/sqlite3_adapter'
```

Comment:
A comment explaining why this monkey patch is necessary would be helpful.

Reply:
This monkey patch is necessary because the `SQLite3Adapter` is smart enough to encode strings that come in with `Encoding::ASCII-8BIT` to `Encoding::UTF-8`. This is what we want all adapters to do but thats not what they all do. So to mimic the worst case, for example using a bad oracle adapter, we monkey patch the `SQLite3Adapter` to just return the string instead of trying to encode it using `utf8`.